### PR TITLE
Cy 2950 add jenkins pipeline

### PR DESCRIPTION
**Pull request by alexmolev**

### CODE REVIEW

File: jenkins/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    buildDiscarder(logRotator(numToKeepStr:'30'))
+    timeout(time: 60, unit: 'MINUTES')
+    timestamps()
+  }
+
+  environment {
+    branch = "master"
+    workspace = "${env.WORKSPACE}"
+    target_dir = "cloudify-java-rest-client-${env.BUILD_NUMBER}"
+    cfy_manager_url = 'https://raw.githubusercontent.com/cloudify-cosmo/cloudify-java-rest-client'
```
Comment (warolv) [inline jenkins/Jenkinsfile:21]: better to call it cloudify-java-rest-client_url

File: jenkins/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  options {
+    checkoutToSubdirectory('cloudify-java-rest-client')
+    buildDiscarder(logRotator(numToKeepStr:'30'))
+    timeout(time: 60, unit: 'MINUTES')
+    timestamps()
+  }
+
+  environment {
+    branch = "master"
```
Comment (warolv) [inline jenkins/Jenkinsfile:18]: It's only for master branch?

File: jenkins/build-prod.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+    - name: mvn
+      image: maven:3.6.3-jdk-8
+      resources:
+        requests:
+          cpu: 0.5
```
Comment (warolv) [inline jenkins/build-prod.yaml:9]: This parameter defines how many pods you can run per VM with 2 cpus, meaning you can run 3 pods(jobs) for one VM. that what you want?

File: jenkins/Jenkinsfile (hunk 2 above)
Comment (alexmolev) [reply jenkins/Jenkinsfile:18]: fixed

File: jenkins/Jenkinsfile (hunk 1 above)
Comment (alexmolev) [reply jenkins/Jenkinsfile:21]: done

File: jenkins/build-prod.yaml (hunk 3 above)
Comment (alexmolev) [reply jenkins/build-prod.yaml:9]: We can tweak it later if it's too either too much or not enough

Comment (warolv) [reply jenkins/build-prod.yaml:9]: ok

**Pull request merged**